Specifications/Compiler/com.apple.compilers.model.coredata.xcspec

```diff
@@ -0,0 +1,74 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+(
```

Comment:
Again, would be great to split this into two files like the existing specifications.